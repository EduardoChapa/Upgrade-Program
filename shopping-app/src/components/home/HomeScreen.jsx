import React from 'react'
import MainBanner from '../ui/banner/MainBanner';

import Section from '../ui/Section';

function HomeScreen() {
  return (
    <div className='homescreen'>
      <MainBanner />
      <Section title='For you'/>
      <Section title='Popular'/>
      <Section title='Hot sales'/>
      <Section title='Yeah'/>
    </div>
  );
}

export default HomeScreen;