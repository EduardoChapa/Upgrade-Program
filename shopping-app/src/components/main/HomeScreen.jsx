import React from 'react'
import Section from './Section';

function HomeScreen() {
  return (
    <div className='homescreen'>
      <Section title='For you'/>
      <Section title='Popular'/>
      <Section title='Hot sales'/>
      <Section title='Yeah'/>
    </div>
  );
}

export default HomeScreen;