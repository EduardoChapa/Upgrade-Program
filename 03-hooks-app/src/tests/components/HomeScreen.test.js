import React from 'react';
import { shallow, mount } from 'enzyme';
import HomeScreen from '../../components/use-context/HomeScreen';
import UserContext from '../../components/use-context/UserContext';

describe('Testing HomeScreen component', () => {

    const user = {
        name: 'Ricardio',
        email: 'sample@mail.com'
    };

    const wrapper = mount(
        <UserContext.Provider value={ { user } } >
            <HomeScreen />
        </UserContext.Provider>
    );

    test('should display correctly ', () => {
        expect(wrapper).toMatchSnapshot();
    });
});