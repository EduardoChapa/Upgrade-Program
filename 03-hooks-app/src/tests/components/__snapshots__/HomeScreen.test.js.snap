// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing HomeScreen component should display correctly  1`] = `
<div>
  <h1>
    Home Screen
  </h1>
  <hr />
  <pre>
    {
   "name": "Ricardio",
   "email": "sample@mail.com"
}
  </pre>
</div>
`;
